### PR TITLE
Create Gameplay stubs (Sliderbreak, Tablet cover, Fingerlock, Mind block)

### PR DESCRIPTION
**Pull request by paper-toaster**
**Creates stubs for the following terms:**

- Sliderbreak
- Tablet cover
- Fingerlock
- Mind block

Part of organization effort #3143.

---
*8/15 - how did i not have an aneurysm writing this?*

### CODE REVIEW

File: wiki/Gameplay/Fingerlock/en.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  - wrist lock
+  - finger-lock
+  - finger lock
+---
+
+# Fingerlock
+
+*Not to be confused with [Notelock](/wiki/Notelock) or [Mind block](/wiki/Mind_block)*
+
+**Fingerlock** (alternatively spelt as `finger-lock` or `finger lock`) is a term used to describe the sensation of momentarily losing control of one's fingers whilst playing osu!. Although not a term specific to osu!, the term is widely used by players to communicate frustration with plays where a player knows, when and where to press a [hit object](/wiki/Hit_object), but unexplicably just doesn't press the key, effectively creating a sensatin where said finger feels as if it is "locked" in place. Hence then name, "fingerlock."
```
Comment (MegaApplePi) [inline wiki/Gameplay/Fingerlock/en.md:13]:
```suggestion
**Fingerlock** (alternatively spelt as `finger-lock` or `finger lock`) is a term used to describe the sensation of momentarily losing control of one's fingers whilst playing osu!. Although not a term specific to osu!, the term is widely used by players to communicate frustration with plays where a player knows, when and where to press a [hit object](/wiki/Hit_object), but inexplicably just doesn't press the key, effectively creating a sensation where said finger feels as if it is "locked" in place. Hence then name, "fingerlock."
```

File: wiki/Gameplay/Fingerlock/en.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Fingerlock is a common sensation that happens to many veteran osu! players. The common consensus amongst most players is that fingerlock is generally no cause for alarm. However, if a player experiences persistent or painful fingerlock often, it is recommended that that player see a medical professional, as it may be the result of an underlying health condition.
+
+A specific cause for fingerlock has neither been found nor generally understood yet. However, there are a few common habits that may cause or increase the chances of fingerlock: long play sessions (especially ones that retry the same [beatmap](/wiki/Beatmaps) repeatedly), bad posture or wrist position, or unnecessary strain on the wrist or fingers for extended periods of time. Regardless, fingerlock will most often occur once or twice during a play session as a result of simple hesitation or slow reaction time.
+
+## Remedies and prevention
```
Comment (Joehuu) [inline wiki/Gameplay/Fingerlock/en.md:27]:
```suggestion
## Remedies & prevention
```

File: wiki/Gameplay/Mind_block/en.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+*Not to be confused with [Notelock](/Beatmapping/Overall_difficulty#notelock) or [Fingerlock](/wiki/Gameplay/Fingerlock).*
+
+**Mind block** is a term used to describe when a player repeatedly fails or misses on what was a previously easy section of a [beatmap](/wiki/Beatmaps), often as a result of retrying said beatmap repeatedly. The term is named "mind block" since the event that it describes creates the sensation that the player's brain (mind) has been figuratively "blocked" from successfully obtaining a full combo or completing the aforementioned section.
+
+## Causes
+
+Mind block is often caused by heavy build-up of "useless [muscle memory](https://en.wikipedia.org/wiki/Muscle_memory "Wikipedia")" from constant, repeated, and quick retrying of a beatmap. Although no official science or research has been conducted on this specific phenomenon in osu!, it is widely believed by the community that during mind block, the player's mind is often bored, burned-out, or tired of doing the same patterns and hearing the same few seconds of music, that it struggles to focus and pay attention, and thus struggles to develop new muscle memory.
+
+## Remedies & prevention
```
Comment (Joehuu) [inline wiki/Gameplay/Mind_block/en.md:22]: or
```suggestion
## Remedies and prevention
```

**Review by Joehuu — COMMENTED** [review]
LGTM after this.

Also not really sure about the consensus of adding a space or not on the default format of these gameplay terms.

**Review by MegaApplePi — APPROVED** [review]

**Pull request merged**